doc/reference/std/misc/decimal.md

```diff
@@ -0,0 +1,232 @@
+# Decimal
```

Comment:
you need to add it to `.vuepress/config.js` as well, so that it appears in the sidebar/hamburger menu on the phone and it is searchable.

Reply:
Also, let's call the title `Decimal Numbers`

Reply:
Done